tools/scylla-nodetool.cc

```diff
@@ -2161,6 +2784,24 @@ Fore more information, see: https://opensource.docs.scylladb.com/stable/operatin
             },
             stop_operation
         },
+        {
+            {
+                "tablestats",
```

Comment:
This has a alias (`cfstats`) which I think is more commonly used. Operations support aliases, see `operation::operation()`, please add the `cfstats` alias to this command.

Reply:
Also, please add at least one tests where you use the alias, to make sure it works as expected.

Reply:
ack. done and done.